### PR TITLE
Bug 1798131: Fix CSI snapshot tests

### PR DESCRIPTION
**Pull request by bertinatto**
CC @openshift/storage

### CODE REVIEW

File: vendor/k8s.io/kubernetes/test/e2e/generated/bindata.go [hunk 1]
```diff
@@ -2278,7 +2278,7 @@ func testE2eTestingManifestsStorageCsiGcePdController_ssYaml() (*asset, error) {
...
...3e\x56\x02\xbb\xc2\x9f\xe0\x9a\x00\x5c\xea\xa3\x82\x59\xc8\xe5\x17\x30\xd1\x6d\x22\x77\x74\xd0\x1e\x26\x70\x7b\x89\x0e\xee\x2a\x7d\x25\xb9\x29\x78\x75\x4e\x8d\xa7\x0b\x76\x6f\xa5\xee\xba\xe4\x73\x65\xba\xdf\xac\xfd\x34\xba\x29\xd5\x03\x3c\x21\xa9\x77\xf4\x50\x05\x62\xc8\x11\x44\xae\x11\xd8\xc2\xab\xf5\x2b\x78\x55\xbc\x84\xf9\xf3\x8c\xc6\x30\x7f\xa6\x31\x2c\x66\x34\x19\x4c\x72\x7e\x93\xc9\xd7\x57\xdc\x15\x23\xcb\x55\x08\x19\xb1\xe0\x70\x00\x74\x21\x8b\x1b\x0c\x29\x4d\x0f\x0d\xac\xb6\x6d\x57\x7c\xae\xc2\xf6\x7a\xf3\x03\x02\xbb\xe0\x0a\xdc\xce\x26\x7c\xf6\x16\x8c\xf9\x75\xa3\x63\x2e\xe9\x2f\x2f\x87\xbd\x25\x3b\x40\x1f\x97\xed\x20\x53\xf2\xc5\x93\xe0\x75\x5f\x01\x83\x4e\xa0\xc7\x2d\xab\xfd\xcc\x1a\xa6\x1a\x6d\x56\xbd\x1d\xe7\x1e\x4e\xd0\x55\x70\xa7\xef\x3e\xd7\x18\x0e\x14\xf0\x21\x73\x70\x5e\xd5\x4a\x63\x51\x62\x01\x73\x5b\xc0\x02\x65\xf0\x8a\xd7\x30\xb7\x5a\xc9\xf5\x18\x9c\xc7\x5a\xd9\x40\x7a\x0d\x05\xbe\x28\x83\x05\xd4\x4a\xc0\x7c\xb3\xae\xc4\xe2\x7f\x24\x9e\x2f\xe6\x87\x45\xff\xe3\xcf\x41\xce\xc3\x91\x6b\x2c\xa6\x53\xb5\xc2\xab\x3a\xb2\xc7\x52\x11\x7b\xd1\xd6\xe1\x8e\xc7\x81\x7f\x61\xca\x48\x9c\xba\x5d\x59\x52\x47\xae\xe7\x51\xfc\x0c\x00\x00\xff\xff\x9f\xb0\xc3\xe5\x78\x12\x00\x00")
```
Comment (bertinatto) [inline vendor/k8s.io/kubernetes/test/e2e/generated/bindata.go]: The diff in this file wasn't part of the upstream PR, but it was created when I ran `hack/update-generated-bindata`.

File: test/extended/testdata/csi/aws-ebs/install-template.yaml [hunk 2]
```diff
@@ -1,7 +1,6 @@
 # Installation of AWS EBS CSI driver into OpenShift.
 # Source: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/e727882eae38614ec3246b243255e536aeef5a0f/deploy/kubernetes/manifest.yaml
 # Tailored for OCP 4.2:
```
Comment (jsafrane) [inline test/extended/testdata/csi/aws-ebs/install-template.yaml]: this should be 'tailored for OCP', without any version.

**Comment by bertinatto** [comment]
```
level=error msg="Error: Error applying IAM policy to project \"openshift-gce-devel-ci\": Too many conflicts.  Latest error: Error setting IAM policy for project \"openshift-gce-devel-ci\": googleapi: Error 409: There were concurrent policy changes. Please retry the whole read-modify-write with exponential backoff., aborted"
level=error
level=error msg="  on ../tmp/openshift-install-267682826/master/main.tf line 26, in resource \"google_project_iam_member\" \"master-service-account-user\":"
level=error msg="  26: resource \"google_project_iam_member\" \"master-service-account-user\" {"
```
/test e2e-gcp-upgrade

**Comment by bertinatto** [comment]
```
level=info msg="Waiting up to 30m0s for the cluster at https://api.ci-op-vix9mndp-db044.origin-ci-int-gce.dev.openshift.com:6443 to initialize..."
level=info msg="Cluster operator insights Disabled is False with : "
level=error msg="Cluster operator openshift-apiserver Degraded is True with APIServerDaemonSet_UnavailablePod: APIServerDaemonSetDegraded: 1 of 3 requested instances are unavailable"
level=fatal msg="failed to initialize the cluster: Cluster operator openshift-apiserver is still updating"
```

/test e2e-gcp-upgrade

**Pull request merged**